index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>collabortion project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p id="tribute-info">The man who saved a billion lives</p>
+        
+        <a href="#" id="tribute-link" target="_blank"></a>
+        <div id="img-div">
+            <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="" id="image">
+            <div id="img-caption">
+                Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+            </div>
```

Comment:
- [ ] What do you think about having a p tag for this caption? Don't you think that would follow the semantic principle and even allow others, such as screen readers to understand? I would suggest you use the p tag instead of the div.